src/components/analytics/__tests__/PatternVisualizationReact.test.tsx

```
@@ -0,0 +1,244 @@
+    confidence: 0.6,
+    correlatedFactors: [{ factor: 'Depression', strength: 0.5 }],
+    significance: 'Medium risk factor',
+  },
+]
+
+// Mock the recharts components to avoid rendering issues in tests
+vi.mock('recharts', () => {
+  const OriginalModule = vi.importActual('recharts')
+  return {
+    ...OriginalModule,
+    ResponsiveContainer: ({ children }: { children: React.ReactNode }) => (
```

ESLint

[error] 'React' is not defined.

(no-undef)